npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -195,6 +208,15 @@ export async function getAngularCliModules (projectRoot: string) {
   }
 }
 
+async function getInstalledPackageVersion (pkgName: string, projectRoot: string): Promise<string> {
+  const packageJsonPath = require.resolve(`${pkgName}/package.json`, { paths: [projectRoot] })
```

Comment:
Just had to double check, but this looks fine - some packages do not expose `package.json`, but this one does: https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/core/package.json#L23